vNext/AISKU/snippet/snippet.js

```diff
@@ -1,96 +1,96 @@
-﻿var sdkInstance = "appInsightsSDK";
-window[sdkInstance] = "appInsights";
-var aiName = window[sdkInstance]; // provide non default instance name through key appInsightsSDK
-var aisdk = window[aiName] || (function (aiConfig) {
-    var appInsights = {
-        config: aiConfig
-    };
-    appInsights.initialize = true; // initialize sdk on download
+﻿(function() {
+    var aiName = "appInsights";
+    var aisdk = window[aiName] || (function (aiConfig) {
+        var appInsights = {
+            config: aiConfig
+        };
+        appInsights.initialize = true; // initialize sdk on download
```

Comment:
Just add this to the declaration above.
var appInsights = {
            initialize: true,
            config: aiConfig
        };